### ISSUE TITLE
`import.meta.url` is `undefined` at Vercel Edge Runtime

### ISSUE THREAD

**Issue by PonomareVlad** [zeit/now#10237]
### Page to Investigate

https://vercel.com/docs/concepts/functions/edge-functions/limitations#many-node.js-apis-are-not-available

### Steps to Reproduce

In section [Many Node.js APIs are not available](https://vercel.com/docs/concepts/functions/edge-functions/limitations#many-node.js-apis-are-not-available) we see this point:
- Use ES modules

But ESM feature `import.meta.url` seems not supported by Vercel Edge Runtime.

**Current behavior**

```js
// api/index.mjs

console.log(import.meta) // {}

console.log(import.meta.url) // undefined
```

At Vercel Edge Runtime `import.meta.url` is `undefined`.

**Expected behavior/code**

Expected specification-driven behavior:

```js
// api/index.mjs

console.log(import.meta) // {url: ".../api/index.mjs"}

console.log(import.meta.url) // ".../api/index.mjs"
```

[Comment from Edge Runtime team](https://github.com/vercel/edge-runtime/issues/486#issuecomment-1643455912)

[Repository to reproduce](https://github.com/PonomareVlad/VercelEdgeURL/tree/main)

**Comment by trek** [comment]
Hi @PonomareVlad ,

Thanks for reaching out! We use this Issue tracker for bugs in the Vercel CLI and related open source libraries. For general platform help and feedback please open a new discussion at https://vercel.community.

**Comment by juyuwol** [comment]
@trek

This is not a "general platform help", but a clear bug report for Vercel CLI. I'm having the same issue.

Until December last year, import.meta.url worked in Edge runtime. At that time, I confirmed that `fetch(file, import.meta.url)` worked, so I switched my serverless Node.js runtime to Edge in my project.

There is the official documentation [Open Graph (OG) Image Generation](https://vercel.com/docs/functions/og-image-generation#edge-runtime) that informs using fetch and import.meta.url to read files in Edge.

That is, import.meta.url worked until some point (at least last year), and this was the actual behavior INTENDED by Vercel. But at some point, it stopped working.

This is a bug.
